js/src/personal/haoking/projects/LeftProjects.tsx

```diff
@@ -1,28 +1,51 @@
-import { Text, Title, Box } from '@mantine/core'
-import { theme } from '../theme.ts'
+import {
+  Text,
+  Title,
+  Box,
+  Flex,
+  Image,
+  useComputedColorScheme,
+} from '@mantine/core'
 import styles from '../Haoking.module.css'
+import { theme } from '../theme.ts'
 
 /* Component for left side of Projects page */
-export function LeftProjects() {
+export function LeftProjects({
+  displayedImageSrc,
+}: {
+  displayedImageSrc: string
+}) {
+  const colorScheme = useComputedColorScheme()
+
   return (
     <Box
       ml={90}
       mt={45}
       className={styles.animate_fade_in}
     >
-      <Title className={styles.title}>
-        <Text
-          inherit
-          variant="gradient"
-          component="span"
-          gradient={{
-            from: theme.colors.black[0],
-            to: theme.colors.black[1],
-          }}
-        >
-          {'Projects'}
-        </Text>
-      </Title>
+      <Flex direction="column">
+        <Title className={styles.title}>
+          <Text
+            inherit
+            variant="gradient"
+            component="span"
+            gradient={
+              colorScheme === 'light' ?
+                { from: theme.colors.accent[2], to: theme.colors.accent[1] }
+              : { from: theme.colors.black[0], to: theme.colors.black[1] }
+            }
+          >
+            {'Projects'}
+          </Text>
+        </Title>
+        <Image
```

Comment:
instead of setting or not setting attributes on the image tag, just don't make the image tag if no image should be shown.